code/index.html

```diff
@@ -1,22 +1,171 @@
+      <h1>Caesar crosses the Rubicon
+        as Rome braces itself for Civil
+        War
+      </h1>
+      <div class="journalist">
+        <div class="journalist-left">
+          <p>By Titus Livius</p>
+          <time>Wed 22 Jan 2020 12:00 GMT</time>
+        </div>
+        <div class="journalist-socials">
+          <a href="#" class="fa fa-envelope"></a>
+          <a href="#" class="fa fa-whatsapp"></a>
+        </div>
+      </div>
+    </section>
+
 
     <section class="big-news">
-      <!-- The big news content -->
+      <figure>
+        <img class="big-image" src="therubicon.jpeg" alt="rubicon river">
+        <figcaption>The river has long served
+          as a natural border between Italy and its province
+          Gaul. Photgraph: Anachronism &copy;
+        </figcaption>
+      </figure>
+
+      <p class="big-text">Julius Caesar has led a single legion,
+        Legio XIII, south over the Rubicon from Cisalpine Gaul to Italy, making his way to Rome.
+        In doing so, he has deliberately broken the law on imperium and made armed conflict
+        inevitable. Sources have described Caesar as undecided as he approached the river, and
+        attribute the crossing to a supernatural apparition. It was reported
+        that Caesar dined with Sallust, Hirtius, Oppius, Lucius Balbus and Sulpicus
+        Rufus on the night after the crossing into Italy on January 10.
+
+        Before crossing, Caesar is reported to have said: "ālea iacta
+        est" ("the die has been cast")<br>Legal commentators note that imperium is forbidden by
+        the law as a capital offence. Furthermore, obeying the commands of a
+        general who does not legally possess imperium is a capital offence. If a
+        general enters Italy in command of an army, both the general and his
+        soldiers become outlaws and are automatically condemned to death. Generals
+        are thus obliged to disband their armies before entering Italy.</p>
+
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <div class="grid-news-header">
+      <h1>Also popular this week:</h1>
+    </div>
+
+    <section class="grid-news-container">
+      <div class="effect-container">
+        <img class="grid-image" src="colloseum.jpeg" alt="The colloseum" />
+        <div class="content">
+          <div class="text">"Bread and Circuses": Emperor Gaius outlines new vision of government</div>
+        </div>
+      </div>
+      <div class="effect-container">
+        <img class="grid-image" src="hannibal.jpeg" alt="state head" />
+        <div class="content">
+          <div class="text">Elephants and their Generals: The Alps 200 years after Hannibal</div>
+        </div>
+      </div>
+      <div class="effect-container">
+        <img class="grid-image" src="library.jpeg" alt="state head" />
+        <div class="content">
+          <div class="text">Alexandria burns in Caesar's wake: locals fear library perished</div>
```

Comment:
This maybe will be more suited as a `<p>`?
```suggestion
          <p>Alexandria burns in Caesar's wake: locals fear library perished</p>
```